ops/k8s/configs/oauth.yaml.ctmpl

```diff
@@ -0,0 +1,21 @@
+{{ with $environment := env "ENVIRONMENT" }}
+{{ with $suffix := env "SUFFIX" }}
+{{ with $project := env "GOOGLE_CLOUD_PROJECT" }}
+{{ with secret (printf "secret/dsde/datarepo/%s/clientid-secret.json" $environment ) }}
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: google-clientid-secret
+data:
+  clientid_secret: {{ .Data.oauthClientSecret | base64Encode }}
```

Comment:
Let's make all of these camelCase

Reply:
are you referring to `clientid_secret`

Reply:
You cannot camelCase the keys in the kube yamls